### ISSUE TITLE
Your API Blueprint needs to be build with Snow Crash 0.9.0 or higher.

### ISSUE THREAD

**Issue by shavo007** [pixelfusion/blueman#20]
I have generated my json file using protagonist. Version 1.2.0 which is the latest.

```

  grunt.registerTask('generateJson', 'Generate js version of referral file.', function () {
    var protagonist = require('protagonist');
    var content = grunt.file.read('../../../../doc/public/referral-api.md');


    var result = protagonist.parseSync(content);
    console.log(result);
    var json = JSON.stringify(result);
    grunt.file.write('test/frontend/referral.json', json);

  });

```

When i run the command to convert I get that exception.
```
blueman convert referral.json 

```

Any ideas?

**Comment by shavo007** [comment]
I have even tried this with drafter and I still get the same result.

Version of drafter is v2.0.0

**Comment by jvanharn** [comment]
I have the same issue. Probably because Snowcrash no longer exists. The versioning is reset, thus bringing the current version below 2.0 to 1.2.0.

**Comment by kielabokkie** [comment]
Hi guys, thanks for reporting this issue. I'll have a look in the next few days and see what I can do to fix it.

**Comment by WolfgangD** [comment]
Stuck on the same issue. Is there a workaround we can use in the meantime?

I'm not that familiar with php but willing to investigate a fix if you have some pointers where to start.

**Comment by WolfgangD** [comment]
I actually fed the wrong json. It is working for me now.

Using `drafter` I create the json for blueman with
`drafter -f json -t ast -o api.json api.apib`

Then I need to manually take what's stored in the "ast" property and use this as input for blueman

`blueman convert api_modified.json`

**Comment by MitchellMcKenna** [comment]
@WolfgangD Thanks for your temporary workaround, worked for me.

**Comment by Ustice** [comment]
The workaround that @WolfgangD proposed doesn't work for me, and I am unable to get past this error. @kielabokkie, were you able to figure out a fix for this issue? I started to investigate, and then I realized that the installer is not a part of the repo, and without being strong in php any more and without build  instructions, I figured that I would check here first.

**Comment by kielabokkie** [comment]
I've released Blueman [v1.2.0](https://github.com/pixelfusion/blueman/releases/tag/1.2.0) which solves this issue.

If you are using the .phar file just update using the `self-update` function:

    $ blueman self-update 

I've also updated the readme to make it clear that Blueman only supports the `AST` format. I read that the `AST` format is no longer officially supported but for now you can still generate `AST` formatted files with Drafter. If anyone want to build in support for the new `refract` format then feel free to do so. Just create a PR and I'll take care of updating the installer.
